plugin.xml

```diff
@@ -22,7 +22,14 @@
         </config-file>
         <header-file src="src/ios/CDVTestFairy.h" />
         <source-file src="src/ios/CDVTestFairy.m" />
-        <framework src="TestFairy" type="podspec" spec="1.28.7" />
```

Comment:
Nice!!! 

"framework" tag with type "podspec" is deprecated and will be removed. Please use the "podspec" tag. 

FYI

https://github.com/apache/cordova-discuss/pull/108